metadata/.gitignore

```diff
@@ -0,0 +1,14 @@
+# These are added during tree generation only.
+/cache
+/dtd
+/glsa
+/md5-cache
+/news
+/herds.xml
+/timestamp
+/timestamp.chk
+/timestamp.x
+/pkg_desc_index
+/projects.xml
+/xml-schema
+/.checksum-test-marker
```

Comment:
I think this .gitignore can completely go.. we will never generate these files afaik